kubernetes/policies/pss/baseline/9_unsafe_sysctl_options_set_test.rego

```diff
@@ -0,0 +1,116 @@
+package appshield.kubernetes.KSV026
+
+test_denied {
```

Comment:
Would change to something more inductive like:
test_sysctls_restricted_property_denied

Reply:
oh I missed that.
fixed